shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -290,26 +291,27 @@ instance NoUnexpectedThunks (EpochState crypto)
 
 instance Crypto crypto => ToCBOR (EpochState crypto)
  where
-  toCBOR (EpochState a s l p n) =
-    encodeListLen 4 <> toCBOR a <> toCBOR s <> toCBOR l <> toCBOR p <> toCBOR n
```

Comment:
Slightly worrying that this was wrong before!

Reply:
indeed